src/lib.rs

```diff
@@ -27,7 +28,7 @@ pub mod uint;
 pub mod writer;
 
 pub mod error;
-mod num;
+pub mod num;
```

Comment:
Why do you need the mod, rather than its enclosed types?

Reply:
You're right, I think we needed this at some point but after a refactor it's not necessary.